src/pointer-analysis/dynamic_object_name.h

```diff
@@ -55,4 +55,6 @@ inline std::string get_dynamic_object_name(
   return name;
 }
 
+const std::string &get_dynamic_object_prefix();
```

Comment:
This should be used in the function above and therefore needs to come before it.